disnake/app_commands.py

```diff
@@ -453,13 +453,20 @@ class ApplicationCommand(ABC):
         Defaults to ``True``.
 
         .. versionadded:: 2.5
+
+    nsfw: :class:`bool`
+        Whether this command is :ddocs:`age-restricted <interactions/application-commands#agerestricted-commands>`.
+        Defaults to ``False``.
```

Comment:
note that bots in discovery **cannot** have nsfw (app) commands.

Reply:
The linked api docs will mention this:
https://github.com/discord/discord-api-docs/blob/766f6c5adf3aa14cc0ee6a7dca5ef3f9532dc534/docs/interactions/Application_Commands.md?plain=1#L1024